app/src/openxr/cpp/OpenXRInputMappings.h

```diff
@@ -189,6 +190,33 @@ namespace crow {
                     { kPathHaptic, OpenXRHandFlags::Both },
             },
     };
+
+    // Pico controller: this definition was created for the Pico 4, but the Neo 3 will likely also be compatible
+    const OpenXRInputMapping Pico4 {
+            "/interaction_profiles/pico/neo3_controller",
```

Comment:
Haven't seen any interaction profile path published by Khronos so far, I guess you're simply making it up?

Reply:
I used information I got from the Pico developers that assist over at Godot, so not totally made up.

That said, Pico is indeed behind on publishing things through Khronos, they claim to be working on amending the OpenXR spec with their controllers and extensions, but I do not know when that will actually happen / get published.